lib/science_wire_harvester.rb

```diff
@@ -443,24 +189,271 @@ def harvest_from_wos_id_cap_profile_id_report(path_to_report)
     process_wos_ids_for_author(batch_profile_id, wos_ids)
   end
 
-  # @param [String] line A string with the format "WOS:000314860700023	38808"
-  # @return [Array<String>] first element wos_id. second_element cap_profile_id
-  def parse_wos_line(line)
-    line =~ /WOS:([^\s]*)\s+(\d+)/
-    [Regexp.last_match(1), Regexp.last_match(2)]
-  end
+  private
 
```

Comment:
While we're moving methods around, let's put them in alphanumeric order by method name.